src/three/loaders/SUBTREELoader.js

```diff
@@ -0,0 +1,694 @@
+/**
+ * Structure almost identical to Cesium, also the comments and the names are kept
+ * https://github.com/CesiumGS/cesium
+ */
+import {DefaultLoadingManager, Matrix3, Matrix4, Vector3} from 'three';
```

Comment:
I still have to dig into this file a bit more and understand the spec but it looks as though like this basically expands implicit tile nodes into a full hierarchy of children. Is that right?

If that's I think ideally this wouldn't rely on three.js. The tiles can be expanded into in a general structure as they typically are in the tileset.json files. And then the existing node preprocessing parse any necessary fields into three.js constructs used during traversal.